### PR TITLE
Pylint enable string checker

### PR DESCRIPTION
**Pull request by glicht**
## Status
Ready

## Description
Until now we were running pylint with errors only. This pr adds also warnings from the string checker except the `duplicate-string-formatting-argument` which seems to be extreme . See: https://docs.pylint.org/en/1.6.0/features.html#string-checker

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Additional changes
Fix the pylint errors in existing integrations.

## Technical writer review
No need.

### CODE REVIEW

**Comment by glicht** [comment]
@yaakovi see Polyswarm validation failure in build: https://circleci.com/gh/demisto/content/33018?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: Integrations/Whois/Whois.py [hunk 1]
```diff
@@ -8285,7 +8285,7 @@ def whois_command():
                 '%d-%m-%Y')
             md['Expiration Date'] = whois_result.get('expiration_date')[0].strftime('%d-%m-%Y')
     except ValueError as e:
-        return_error('Date could not be parsed. Please check the date again.\n{error}'.format(type(e)))
+        return_error('Date could not be parsed. Please check the date again.\n{}'.format(e))
```
Comment (Itay4) [inline Integrations/Whois/Whois.py]: i think this might cause issues cause of unicode
better to format as `str(e)`

Comment (glicht) [reply Integrations/Whois/Whois.py]: It's the same. `format` internally does the same as `str` to the passed object.

**Comment by glicht** [comment]
Not sure about pylint score. It will be 10, because we disable all non relevant stuff and enforce to fix the errors.

**Pull request merged**